<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthWindow</class>
 <widget class="QMainWindow" name="AuthWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>352</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AuthWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="navBar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#navBar
{
	background-color: white;
}



QPushButton
{
    background-color: #BFFF00;
    color: black;
	border-bottom-left-radius: 10px;
	border-bottom-right-radius: 10px;
	padding: 15px;
}
QPushButton:hover:!pressed
{
	background-color: #99CC00;
}
QPushButton:pressed
{
	background-color: #CCFF33;
	color: black;
}
QPushButton:disabled
{
	background-color: #3F3F3F;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="navAuthButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Fira Sans Condensed</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	border-bottom: 3px solid black;
	border-left: 3px solid black;
	border-right: 3px solid black;
}</string>
         </property>
         <property name="text">
          <string>Авторизация</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="navAboutButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Fira Sans Condensed</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Об игре</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="navRulesButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Fira Sans Condensed</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Правила</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="navFaqButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Fira Sans Condensed</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ЧаВо (FAQ)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="pageBar">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="authPage">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(238, 238, 236);</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>150</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QFrame" name="authFrame1">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">#authFrame1
{
	border: 10px solid black;
	border-radius: 30px;
}</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>25</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QFrame" name="authFrame2">
                <property name="font">
                 <font>
                  <pointsize>1</pointsize>
                 </font>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <property name="spacing">
                  <number>10</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item alignment="Qt::AlignVCenter">
                  <widget class="QLabel" name="authLabel">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="font">
                    <font>
                     <family>Fira Sans Condensed</family>
                     <pointsize>44</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QLabel
{
	color: black;
}</string>
                   </property>
                   <property name="text">
                    <string>Авторизация</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="authLineEdit">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>Fira Sans Condensed Medium</family>
                     <pointsize>24</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QLineEdit
{
	color: #212529;
	padding: 5px;
	border: 3px solid #616161;
	border-radius: 5px;
	background-color: white;
}

QLineEdit:focus
{
	border: 4px solid #272727;
}

QLineEdit:disabled
{
	background-color: #3F3F3F;
	color: #FFF;
}</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="maxLength">
                    <number>10</number>
                   </property>
                   <property name="echoMode">
                    <enum>QLineEdit::Password</enum>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                   <property name="placeholderText">
                    <string>Код</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="authRequestButton">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>Fira Sans Condensed Medium</family>
                     <pointsize>24</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QPushButton
{
    background-color: #BFFF00;
    color: black;
	padding: 15px;
	border: 3px solid black;
	border-radius: 10px;
}
QPushButton:hover:!pressed
{
	background-color: #99CC00;
}
QPushButton:pressed
{
	background-color: #CCFF33;
	color: black;
}
QPushButton:disabled
{
	background-color: #3F3F3F;
}</string>
                   </property>
                   <property name="text">
                    <string>Войти</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>26</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>150</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="aboutPage">
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>70</y>
             <width>611</width>
             <height>131</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>32</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Тут пока ничего нет</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="rulesPage">
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>90</y>
             <width>611</width>
             <height>131</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>32</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Тут тоже ;c</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="faqPage">
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>80</y>
             <width>631</width>
             <height>131</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Будто здесь что-то изменилось бы</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
